code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -149,8 +149,8 @@
 	var/attack_delay = 0 //Bonus or pen to time in between attacks. + makes slashes slower.
 	var/speed = -0.5 //Regular xeno speed modifier. Positive makes you go slower. (1.5 is equivalent to FAT mutation)
 	var/speed_modifier = 0 //Speed bonus/penalties. Positive makes you go slower.
-	var/slowdown = 0 //Temporary penalty on movement. Regenerates each tick.
-	var/stagger = 0 //Temporary inability to use special actions. Regenerates each tick.
+	slowdown = 0 //Temporary penalty on movement. Regenerates each tick.
+	stagger = 0 //Temporary inability to use special actions. Regenerates each tick.
```

Comment:
These are no longer necessary here. The comments can be moved to the variable definition, with the stagger explaining also the effect on humans.

Reply:
Fucking knew you were going to harp on this.